### ISSUE TITLE
gems unavailable in git bash

### ISSUE THREAD

**Issue by mxriverlynn** [vertiginous/pik#63]
i installed and configured v0.2.8 in my git bash shell, and it basically works... except gems are not available.

i can call 'pik use 187' and it changes to v1.8.7... then when i call 'gem list' it shows there are no gems available.

if i do the same thing from a standard command windows, it shows the gems i have installed, correctly.

**Comment by vertiginous** [comment]
I see this too.  I'll look into it.

**Comment by vertiginous** [comment]
pik's not unsetting $GEM_HOME and $GEM_PATH properly.

```
~/Work/repo/pik > pik 1.9.2

~/Work/repo/pik > gem list

*** LOCAL GEMS ***



~/Work/repo/pik > unset GEM_HOME

~/Work/repo/pik > unset GEM_PATH

~/Work/repo/pik > gem list

*** LOCAL GEMS ***

abstract (1.0.0)
  ... all my gems ... 
ZenTest (4.3.2, 4.3.1)
```

I'll have this fixed in the next release.  Sorry about that.

**Comment by mxriverlynn** [comment]
cool... easy work around for now is to add 

<pre><code>unset GEM_HOME
unset GEM_PATH
</code></pre>

to my <code>.bashrc</code> file.

**Comment by vertiginous** [comment]
Yeah, if you're not using GEM_HOME at all, the best way to fix it would probably be to modify ~/.pik/.pikrc to 

```
#!/bin/sh
pik_path=/c/bin

function pik  {
  $pik_path/pik_runner.exe pik.sh $@
  [[ -s $USERPROFILE/.pik/pik.sh ]] && source $USERPROFILE/.pik/pik.sh
  unset GEM_HOME
  unset GEM_PATH
} 
```

**Comment by briandonahue** [comment]
I'm seeing this same issue and I just installed the pik gem today.  Is this fixed in a release, and if so, how can I install the fix?

Thanks!

**Comment by luislavena** [comment]
@briandonahue did you modify `~/.pik/.pikrc` file as described by Gordon?

**Comment by cuzic4n** [comment]
same issue.. did the ~/.pik/.pikrc thing and that fixed it.. make sure you source that back in or exit git bash and come back..

**Comment by jwebcat** [comment]
worked for me too edited .pikrc  and its working like a charm.

**Comment by spccdt** [comment]
Gordon's workaround was exactly what I needed, too. Thanks.
